### ISSUE TITLE
could it support a nested structure?

### ISSUE THREAD

**Issue by Morriz** [Bhoos/react-native-theming#12]
Hi, I would love it if I can do this:
```
createStyle({
  someModule: {
    someStyle: 'red'
  },
  anotherModule: ...
}
```
and then refer to it like '@someModule.someStyle'. As I have quite some styles grouped like that it would be hard to flatten into a list.

**Comment by jonathanroze** [comment]
Pretty good question, any news about that ?

**Comment by dapicester** [comment]
This seems a duplicate of #14, actually solved in #31.
